numpy/core/getlimits.py

```diff
@@ -513,7 +513,7 @@ def __init__(self, int_type):
         self.bits = self.dtype.itemsize * 8
         self.key = "%s%d" % (self.kind, self.bits)
         if self.kind not in 'iu':
-            raise ValueError("Invalid integer data type.")
+            raise ValueError("Invalid integer data type %r." % (self.kind,))
```

Comment:
Looks harmless at first glance if it helps a downstream project. I don't think we have plans to explicitly support Python 3.4 issues moving forward, but this may be fair enough.

I suppose one could add a unit test that uses a regex on the ValueError message, although maybe that's overkill.

Reply:
FWIW those failures also happen with 3.5 at times (so I was told), or I misunderstood and this code path is 3.4 specific?

Reply:
If the issue isn't 3.4-specific that probably make a more compelling case to fix it